src/code/z_camera.c

```diff
@@ -1443,10 +1533,9 @@ s32 Camera_Normal1(Camera* camera) {
     f32 rate = 0.1f;
 
     playerHeight = Player_GetHeight(camera->player);
-    if (RELOAD_PARAMS(camera) || R_RELOAD_CAM_PARAMS) {
+    if (RELOAD_PARAMS(camera) || CAM_DEBUG_RELOAD_PARAMS) {
```

Comment:
I am surprised `CAM_DEBUG_RELOAD_PARAMS` is true in non-debug, so the condition always passes

Is `RELOAD_PARAMS` entirely optimized out then?
Is it usually required for matching? If not maybe there's further refactoring of this conditional that could be done in some way

Reply:
I tried replacing this with `if (1)` everywhere. One benefit is that Camera_KeepOn4 currently has weird match of removing a PRINTF, which would not be necessary if we used `if (1)` instead of `if (RELOAD_PARAMS(camera) || R_RELOAD_CAM_PARAMS)`. However, Camera_Jump3 writes `if (RELOAD_PARAMS(camera) || modeSwitch || R_RELOAD_CAM_PARAMS)` instead of `if (RELOAD_PARAMS(camera) || R_RELOAD_CAM_PARAMS)`, and replacing that with `if (1)` didn't work (neither did rearranging the condition so we could write `modeSwitch || DEBUG_RELOAD_PARAMS(camera)` or something instead). So I went with the `|| CAM_DEBUG_RELOAD_PARAMS` so that case could be consistent too, but I'm not sure if it's worth the weird match in Camera_KeepOn4